frontend/src/components/UpgradeDialog.tsx

```diff
@@ -0,0 +1,112 @@
+import { GemIcon, LifeBuoy, Mail, RefreshCw, Users, Zap } from "lucide-react";
```

Comment:
We will need to remove the old SupportAlby flow right?

Reply:
Not sure I get your point? :thinking:

Reply:
Old comment - you now decided to keep it and allow user to either get pro or support via v4v :raised_hands: